mlflow/system_metrics/metrics/gpu_monitor.py

```diff
@@ -47,5 +47,12 @@ def collect_metrics(self):
             device_utilization = pynvml.nvmlDeviceGetUtilizationRates(handle)
             self._metrics[f"gpu_{i}_utilization_percentage"].append(device_utilization.gpu)
 
+            power_watts = pynvml.nvmlDeviceGetPowerUsage(handle)
```

Comment:
```suggestion
            power_milliwatts = pynvml.nvmlDeviceGetPowerUsage(handle)
```

Reply:
Thanks Haru, yes that's milliwatts.

Reply:
Got it, can we rename this variable to avoid confusion?